bitclock-web/src/libs/gatt.ts

```diff
@@ -7,6 +7,7 @@ let CHR_OTA_SIZE_UUID = "c839db31-c8e7-4c59-938d-c4b46e7bc2a4";
 let CHR_OTA_DATA_UUID = "ed900bcc-2ed5-40b0-b33a-fe7483f254c7";
 
 let CHR_WEATHER_PATH_UUID = "32efee1d-9566-47e2-b880-0026d48c1f81";
+export const CHR_MQTT_URL_UUID = "2ab48c15-934d-11ec-b90a-802748c27b9e";
```

Comment:
Probably best to use `const` instead of `let` but if you do that can you make it consistent here? And export at the bottom.

Reply:
will do for all